integration-libs/cdc/user-profile/register/cdc-register-component.service.spec.ts

```diff
@@ -38,6 +38,9 @@ const userRegisterFormData: UserSignUp = {
   preferences: {},
 };
 
+// Disable unhandled error logging
+config.onUnhandledError = () => {};
```

Comment:
[MEDIUM]
I'm believe that setting this flag in the _global context_ might affect also behavior of other test suites that run after this test suite. I believe changing the global rxjs settings in `beforeEach/All` and reseting to original value in `afterEach/All` would be more bullet-proof and not affect other test suites which run in the same shared global JS enviroment.

Anyway, I'm wondering whether we need to set this config at all, or we can adjust our tests to behave better? Did you consider any alternative solutions, that don't require setting this global rxjs flag?

Reply:
While I agree, that the config change can be moved to `beforeEach/afterEach` for better safety, I'm wondering what value-added brings adjusting tests to behave better.
I followed the trend from the previous PRs containing RxJS upgrade changes and there are more usages of `config.onUnhandledError` in our code, I suggest addressing this (if needed) in a separate ticket.

Reply:
> what value-added brings adjusting tests to behave better. 
Consistency of tests results. Test suites should not interfere with each other implicitly. E.g. if I run other test suite in isolation (for debugging purposes), I might get errors that I wouldn't get if I run all tests alltogether

If the pattern of using `config.onUnhandledError` is common in the epic branch, it sounds reasonable to fix it in a separate PR.

Reply:
We can also add a comment that such config is used only for already existing tests and we shouldn't use it for new tests.

Reply:
To be sure we're on the same page - I agree it's better to move such a config to `beforeEach/afterEach` or similar to not change the global scope and I see an added value for this. What I'm wondering about is what added value brings to adjusting already existing unit tests to avoid setting such a config. Let's explore this in a separate task.

Reply:
Yup. there are 2 levels of changes:
1) SHOULD: move the global rxjs config setting (and resetting) to `before/AfterEach`
2) COULD: investigate if we can get rid of setting (and resetting) this config at all - maybe by adapting a little bit the tests bodies.